diff --git a/lwt/dune b/lwt/dune
index b11ed4d..3d13b2d 100644
--- a/lwt/dune
+++ b/lwt/dune
@@ -2,4 +2,4 @@
  (name cstruct_lwt)
  (wrapped false)
  (public_name cstruct-lwt)
- (libraries cstruct lwt.unix))
+ (libraries cstruct))
diff --git a/lwt/dune b/lwt/dune
index b11ed4d..e59c63a 100644
--- a/lwt/dune
+++ b/lwt/dune
@@ -2,4 +2,4 @@
  (name cstruct_lwt)
  (wrapped false)
  (public_name cstruct-lwt)
- (libraries cstruct lwt.unix))
+ (libraries cstruct lwt))
