diff --git a/lwt/lwt_cstruct.mli b/lwt/lwt_cstruct.mli
index ee0c6eb..2be6600 100644
--- a/lwt/lwt_cstruct.mli
+++ b/lwt/lwt_cstruct.mli
@@ -17,12 +17,12 @@
 
 (** Blocking Lwt functions to read and write from Cstruct buffers. *)
 
-val read: Lwt_unix.file_descr -> Cstruct.t -> int Lwt.t
+(* val read: Lwt_unix.file_descr -> Cstruct.t -> int Lwt.t *)
 (** [read fd t] reads data from the file descriptor [fd] into the
     [t] cstruct.
     @return the numbers of bytes actually read. *)
 
-val write: Lwt_unix.file_descr -> Cstruct.t -> int Lwt.t
+(* val write: Lwt_unix.file_descr -> Cstruct.t -> int Lwt.t *)
 (** [write fd t] writes data from the [t] cstruct to the file
     descriptor [fd].
     @return the numbers of bytes actually written. *)
@@ -35,10 +35,10 @@ val complete: (Cstruct.t -> int Lwt.t) -> Cstruct.t -> unit Lwt.t
     @raise End_of_file if the file descriptor operation fails.
   *)
 
-val sendto: Lwt_unix.file_descr -> Cstruct.t -> Unix.msg_flag list -> Unix.sockaddr -> int Lwt.t
+(* val sendto: Lwt_unix.file_descr -> Cstruct.t -> Unix.msg_flag list -> Unix.sockaddr -> int Lwt.t *)
 (** [sendto fd t flags sa] invokes {!Lwt_unix.sendto} on the [t] cstruct.
     @return the number of bytes sent. *)
 
-val recvfrom: Lwt_unix.file_descr -> Cstruct.t -> Unix.msg_flag list -> (int * Unix.sockaddr) Lwt.t
+(* val recvfrom: Lwt_unix.file_descr -> Cstruct.t -> Unix.msg_flag list -> (int * Unix.sockaddr) Lwt.t *)
 (** [recvfrom fd t flags sa] invokes {!Lwt_unix.recvfrom} on the [t] cstruct.
     @return the number of bytes read and the socket address of the remote side. *)
